lms/templates/dashboard/_dashboard_xseries_info.html

```diff
@@ -28,6 +28,7 @@
             %>
             <a class="btn ${xseries_btn_class}" href="${program_data['program_marketing_url']}" target="_blank"
                data-program-id="${program_data['program_id']}" >
+                <span class="sr">${program_data['display_name']}</span>
```

Comment:
@clytwynec I will add it in code. Actually,  I am not familiar how to create an XSeries in a sandbox. After making the settings on sandbox  I will let you know.